schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,623 @@
+            "properties": {
+              "abstract": {
+                "type": "string",
+                "minLength": 1
+              },
+              "language": {
+                "const": "en"
+              }
+            },
+            "additionalProperties": false
+          }
+        ],
+        "additionalItems": {
+          "type": "object",
+          "properties": {
+            "abstract": {
+              "type": "string",
+              "minLength": 1
+            },
+            "language": {
+               "allOf": [
+                 {"$ref": "#/definitions/language"},
+                 {"not": {"enum": ["en"]}}
+             ]
+            }
+          },
+          "additionalProperties": false
+        }
+      },
+      "affiliation": {
+        "title": "Affiliation",
+        "description": "Institutional affiliation.",
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "The complete name of the affiliation.",
+            "type": "string",
+            "minLength": 1
+          },
+          "ror_id": {
+            "title": "ROR ID",
+            "description": "Research Organization Registry ID.",
+            "type": "string",
+            "pattern": "^https://ror\\.org/0\\w{6}\\d{2}$",
+            "examples": [
+              "https://ror.org/05bnh6r87",
+              "https://ror.org/05gzmn429",
+              "https://ror.org/01ggx4157",
+              "https://ror.org/03c3r2d17"
+            ]
+          }
+        },
+        "required": [
+          "name"
+        ],
+        "additionalProperties": false
+      },
+      "category": {
+        "title": "arXiv category",
+        "description": "This is the abbreviated form of the arXiv category using the category identifier",
+        "type": "string",
+        "pattern": "^([a-z]{3,}-[a-z]{2,}|[a-z]{2,}-?[a-z]+\\.[A-Z]{2})$",
+        "$comment": "TODO: enumeration"
+      },
+      "content_type": {
+        "title": "Content type",
+        "$comment": "TODO: enumeration needs further review, and will need to decide whether the items require their own definitions with more structure.",
+        "type": "string",
+        "enum": ["article", "autobiography", "biography", "book", "book chapter", "book review", "conference proceeding", "conference proceeding index", "erratum", "memorial", "thesis", "lecture notes", "whitepaper"]
+      },
+      "collaboration": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "description": "The fullest representation of the collaborations's name",
+            "type": "string",
+            "minLength": 1
+          }
+        }
+      },
+      "comment": {
+        "title": "Comment",
+        "type": "string",
+        "minLength": 1
+      },
+      "doi": {
+        "title": "DOI",
+        "description": "Digital Object Identifier.",
+        "type": "string",
+        "pattern": "^10\\.\\d{4,9}/\\S+$",
```

Comment:
I don't think this matches all the DOI that are currently in arXiv. I recall an issue where there was a DOI that didn't start with 10.

Reply:
If it was something other than 10 I'm guessing it's part of some other handle system namespace that isn't DOI.

Reply:
This is the issue: https://arxiv-org.atlassian.net/browse/ARXIVNG-3523?jql=text%20~%20DOI%20AND%20project%20%3D%20arXiv-NG%20ORDER%20BY%20status%20DESC

Reply:
It looks like this is in the handle namespace, so not technically a DOI:
> 21 for handles issued by the German Gesellschaft für wissenschaftliche Datenverarbeitung mbH Göttingen (GWDG), the scientific computing center of the University of Göttingen

(from [here](https://en.wikipedia.org/wiki/Handle_System))

Reply:
We have a couple, just ignore them?
count(*) where doi not like '10%';
25

Reply:
I've commented about this in the google doc. DOIs are more general, and can contain multiple dots in the prefix.

Reply:
I've relaxed the pattern.